Resources/Prototypes/_NF/Shipyard/sparrow.yml

```diff
@@ -25,5 +35,4 @@
           overflowJobs: []
           availableJobs:
             StationEngineer: [ 0, 0 ]
-            Scientist: [ 0, 0 ]
-            Borg: [ 0, 0 ]
+            Scientist: [ 0, 0 
```

Comment:
Womp womp, bracket missing, can't buy ship.

Reply:
Maybe it is for the best.
But I fixed it, so we'll never know.